src/memory/transfer_context.cpp

```diff
@@ -121,7 +121,10 @@ void TransferContext::icall(JNIEnv* rawEnv, jobject instance, jlong j_ptr, jint
     JVM_CRASH_COND_MSG(args_size > VARIANT_ARG_MAX, vformat("Cannot have more than %s arguments for method call.", VARIANT_ARG_MAX));
 #endif
 
-    auto* ptr {reinterpret_cast<Object*>(static_cast<uintptr_t>(j_ptr))};
+    Object* ptr;
```

Comment:
Is the check necessary ? j_ptr is only going to be null in the case of a static method, which ignore the point argument anyway.